audioswitch/src/main/java/com/twilio/audioswitch/android/Logger.kt

```diff
@@ -0,0 +1,31 @@
+package com.twilio.audioswitch.android
+
+import android.util.Log
+import com.twilio.audioswitch.BuildConfig
+
+internal class Logger(var loggingEnabled: Boolean = BuildConfig.DEBUG) {
```

Comment:
Yeah I might have gotten carried away with the logger. This is okay, although it is nice to have the unit test coverage here